starkjs/src/compressor12/main_compressor12_exec.js

```diff
@@ -19,6 +19,11 @@ const argv = require("yargs")
     .alias("m", "commit")
     .argv;
 
+
+//# generate the pil files and  const polynomicals files
+//# input files :  $C12_VERIFIER.r1cs  $C12_VERIFIER.const  $C12_VERIFIER.pil
```

Comment:
the inputs here include r1cs and it's input, c12 pil.  And output the const file and exec file (this means that comments in the script are not correct), the process basically likes

1. The r1cs is the constraints of the Stark Verifier, and will be converted to Plonk gate;
2. Generate the Pil code for the Plonk gate;
3. Generate the const polynomial and commit polynomial for the Pil code. but here it does not output all the commit directly, cause it still need the c12a.pil to contrain somes computation, like poseidon, fft etc. 

More detail can be checked at page 14 of this doc, https://github.com/0xPolygonHermez/zkevm-techdocs/blob/main/proof-recursion/v.1.1/proof-recursion.pdf